src/main/java/racing/domain/RacingCars.java

```diff
@@ -0,0 +1,59 @@
+package racing.domain;
+
+import racing.domain.RacingCar;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCars {
+    private static final String NAME_DELIMITER = ",";
+    private final List<RacingCar> racingCarList;
+
+    public RacingCars(String names) {
+        if (isNullOrBlank(names)) {
+            throw new IllegalArgumentException();
+        }
+
+        this.racingCarList = Arrays.stream(toNameArray(names))
+                .map(RacingCar::new)
+                .collect(Collectors.toList());
+    }
+
+    public RacingCars(List<RacingCar> racingCarList) {
+        this.racingCarList = racingCarList;
+    }
+
+    public List<RacingCar> getRacingCarList() {
+        return racingCarList;
+    }
+
+    public int size() {
+        return racingCarList.size();
+    }
+
+    public int getMaxPosition() {
+        return Collections.max(
+                racingCarList.stream()
+                        .map(RacingCar::getPosition)
+                        .collect(Collectors.toList()));
+    }
+
+    public List<String> getWinnerNames() {
+        int highScore = getMaxPosition();
+
+        return racingCarList.stream()
+                .filter(car -> car.isWinner(highScore))
+                .map(RacingCar::getName)
+                .collect(Collectors.toList());
+    }
+
+    private boolean isNullOrBlank(String name) {
```

Comment:
```suggestion
    private boolean isNullOrBlank(String name) {
```

내부 구현 세부 사항을 메소드 이름으로 노출 시키면 어떠한 문제가 있을까요?
구현 로직이 변경되면 메소드이름도 변경해주는게 좋은 패턴일지 고민해보아요~

> 이름을 일반화 하여 표현하기